test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -1692,11 +1692,16 @@ public void PackCommand_OutputDirectorySuffixes(string suffix)
             {
```

Comment:
Also, what happens when we pass the "wrong" slash? Like a backslash `\` on Linux/Mac or a forward slash `/` on Windows?